### PR TITLE
Azure v2 upgrade

### PR DESCRIPTION
**Pull request by Rohi2305**
Created an Article for Azure V1 to V2 Migration.

### CODE REVIEW

File: content/how-to/rackspace-fabric/azure-v2-upgrade/index.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+title: "Customer Experience | Azure V1 to V2 Migration"
+created_date: '2022-06-07'
+created_by: Rohit Rajput
+last_modified_date: '2022-06-08'
+last_modified_by: Rohit Rajput
+product: Rackspace Fabric
+product_url: rackspace-fabric
+---
+
+Customer Experience | Azure V1 to V2 Migration
```
[the remark this answers is not in the document]
Comment (Rohi2305) [reply content/how-to/rackspace-fabric/azure-v2-upgrade/index.md:14]: Updated

File: content/how-to/rackspace-fabric/azure-v2-upgrade/index.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+=======================================
+
+**Follow the below procedure to access Azure Accounts**
```
[the remark this answers is not in the document]
Comment (Rohi2305) [reply content/how-to/rackspace-fabric/azure-v2-upgrade/index.md]: Updated the changes.

File: content/how-to/rackspace-fabric/azure-v2-upgrade/index.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+**Step 3**. In the top navigation panel, click **My Accounts** >> **Rackspace Dedicated**.
+
+<img width="279" alt="Rackspace Dedicated" src="Rackspace Dedicated.png">
+
+
+Managing Permissions
+-----------------------------
+
+**Follow the below procedure to Manage Permissions**
```
[the remark this answers is not in the document]
Comment (Rohi2305) [reply content/how-to/rackspace-fabric/azure-v2-upgrade/index.md]: Updated the changes.
Also changed the Switch into Dedicated Account as well.

**Review by ssalinas24 — CHANGES_REQUESTED** [review]
Bejamin's email is showing in content/how-to/rackspace-fabric/azure-v2-upgrade/Issue Details.png. So take a look at that.

Overall it seems like the new images are not linked from within the file.

File: content/how-to/rackspace-fabric/azure-v2-upgrade/index.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+last_modified_date: '2022-07-01'
+last_modified_by: Rohit Rajput
+product: Rackspace Fabric
+product_url: rackspace-fabric
+---
+
+Azure V2 Upgrade
+=======================================
+
+**How to access Azure Subscriptions**
```
Comment (ssalinas24) [inline content/how-to/rackspace-fabric/azure-v2-upgrade/index.md:17]:
```suggestion
## How to access Azure Subscriptions
```

File: content/how-to/rackspace-fabric/azure-v2-upgrade/index.md [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+
+<img width="366" alt="Microsoft Azure" src="Microsoft Azure.png">
+
+**Step 6.** Under **Microsoft Azure**, you can see the list of **Azure Subscriptions**. 
+
+<img width="470" alt="Azure Subscriptions" src="Azure Subscriptions.png">
+
+
+Accessing Dedicated Hosting Portal
+--------------------------------------------
```
Comment (ssalinas24) [inline content/how-to/rackspace-fabric/azure-v2-upgrade/index.md:45]:
```suggestion
## Accessing Dedicated Hosting Portal
```

File: content/how-to/rackspace-fabric/azure-v2-upgrade/index.md [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+
+**Note**: The Dedicated Hosting Portal (MyRackspace) will only be accessible when the user is currently in a Dedicated Hosting Account.
+
+**How to switch into Dedicated Hosting Account**
```
Comment (ssalinas24) [inline content/how-to/rackspace-fabric/azure-v2-upgrade/index.md:49]:
```suggestion
### How to switch into Dedicated Hosting Account
```

File: content/how-to/rackspace-fabric/azure-v2-upgrade/index.md [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+
+<img width="470" alt="Switch Account for Dedicated updated" src="Switch Account for Dedicated updated.png">
+
+**Step 3.** In the top navigation panel, click **My Accounts** >> **Rackspace Dedicated**.
+
+<img width="349" alt="Rackspace Dedicated" src="Rackspace Dedicated.png">
+
+
+Managing Permissions
+-----------------------------
```
Comment (ssalinas24) [inline content/how-to/rackspace-fabric/azure-v2-upgrade/index.md:65]:
```suggestion
## Managing Permissions
```

File: content/how-to/rackspace-fabric/azure-v2-upgrade/index.md [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+
+**How to manage Azure Permissions**
```
Comment (ssalinas24) [inline content/how-to/rackspace-fabric/azure-v2-upgrade/index.md:67]:
```suggestion
### How to manage Azure Permissions
```

File: content/how-to/rackspace-fabric/azure-v2-upgrade/index.md [hunk 9]
```diff
@@ -0,0 +1,171 @@
...
+<img width="527" alt="Manage Groups User Groups" src="Manage Groups User Groups.png">
+
+**Step 4.** **Select/Deselect** the **Checkboxes** to manage the members of the User Groups and click **Save Changes**.
+
+<img width="306" alt="Edit Group 1" src="Edit Group 1.png">
+
+
+
+**Managing Tickets for Azure v2**
+====================================
```
Comment (ssalinas24) [inline content/how-to/rackspace-fabric/azure-v2-upgrade/index.md:96]:
```suggestion
## Managing Tickets for Azure v2
```

File: content/how-to/rackspace-fabric/azure-v2-upgrade/index.md [hunk 10]
```diff
@@ -0,0 +1,171 @@
...
+
+**You can View, Update and Create tickets by following the procedures below**:
+
+
+**Viewing Azure v2 Tickets**
+----------------------------------------
```
Comment (ssalinas24) [inline content/how-to/rackspace-fabric/azure-v2-upgrade/index.md:102]:
```suggestion
### Viewing Azure v2 Tickets
```

File: content/how-to/rackspace-fabric/azure-v2-upgrade/index.md [hunk 11]
```diff
@@ -0,0 +1,171 @@
...
+
+**How to view Azure v2 Tickets**
```
Comment (ssalinas24) [inline content/how-to/rackspace-fabric/azure-v2-upgrade/index.md:104]:
```suggestion
#### How to view Azure v2 Tickets
```

File: content/how-to/rackspace-fabric/azure-v2-upgrade/index.md [hunk 12]
```diff
@@ -0,0 +1,171 @@
...
+
+**Step 4.** The **Ticket List** opens and you can view either the most recently created ticket or last modified ticket at the top. 
+
+**Note: For additional functionality and features, please click [here](https://docs.rackspace.com/support/how-to/fabric-ticketing) to review the Fabric Ticketing overview document.**
+
+<img width="220" alt="View Ticket" src="View Ticket.png">
+
+
+
+**Creating a Ticket for Azure v2 Subscriptions**
```
Comment (ssalinas24) [inline content/how-to/rackspace-fabric/azure-v2-upgrade/index.md:126]:
```suggestion
### Creating a Ticket for Azure v2 Subscriptions
```

File: content/how-to/rackspace-fabric/azure-v2-upgrade/index.md [hunk 13]
```diff
@@ -0,0 +1,171 @@
...
+------------------------------------
+
+**How to Create a Ticket**
```
Comment (ssalinas24) [inline content/how-to/rackspace-fabric/azure-v2-upgrade/index.md:129]:
```suggestion
#### How to Create a Ticket
```

File: content/how-to/rackspace-fabric/azure-v2-upgrade/index.md [hunk 14]
```diff
@@ -0,0 +1,171 @@
...
+**Step 4.** The **Ticket List** opens and you can view either the most recently created ticket or last modified ticket at the top. 
+
+**Note: For additional functionality and features, please click [here](https://docs.rackspace.com/support/how-to/fabric-ticketing) to review the Fabric Ticketing overview document.**
+
+<img width="220" alt="View Ticket" src="View Ticket.png">
+
+
+
+**Creating a Ticket for Azure v2 Subscriptions**
+------------------------------------
```
Comment (ssalinas24) [inline content/how-to/rackspace-fabric/azure-v2-upgrade/index.md:127]:
```suggestion
```

**Review by ssalinas24 — CHANGES_REQUESTED** [review]

File: content/how-to/rackspace-fabric/azure-v2-upgrade/index.md [hunk 15]
```diff
@@ -0,0 +1,166 @@
...
+created_date: '2022-06-07'
+created_by: Rohit Rajput
+last_modified_date: '2022-07-01'
+last_modified_by: Rohit Rajput
+product: Rackspace Fabric
+product_url: rackspace-fabric
+---
+
+# Azure V2 Upgrade
+=======================================
```
Comment (ssalinas24) [inline content/how-to/rackspace-fabric/azure-v2-upgrade/index.md:15]:
```suggestion
```

**Review by ssalinas24 — CHANGES_REQUESTED** [review]

**Review by ssalinas24 — APPROVED** [review]

**Pull request merged**